openmc/mgxs/library.py

```diff
@@ -382,9 +382,7 @@ def get_mgxs(self, domain, mgxs_type):
         ----------
         domain : Material or Cell or Universe or Integral
             The material, cell, or universe object of interest (or its ID)
-        mgxs_type : {'total', 'transport', 'absorption', 'capture', 'fission',
-                     'nu-fission', 'scatter', 'nu-scatter', 'scatter matrix',
-                     'nu-scatter matrix', 'chi'}
+        mgxs_type : {'total', 'transport', 'absorption', 'capture', 'fission', 'nu-fission', 'scatter', 'nu-scatter', 'scatter matrix', 'nu-scatter matrix', 'chi'}
```

Comment:
Sure, good to know. I must have missed that error when building the html myself.